docs/content/http-api-authorization.md

```diff
@@ -23,40 +24,93 @@ This tutorial requires [Docker Compose](https://docs.docker.com/compose/install/
 
 ## Steps
 
-### 1. Bootstrap the tutorial environment using Docker Compose.
+### 1. Create a policy bundle.
 
-First, create a `docker-compose.yml` file that runs OPA and the demo web server.
+Create a policy that allows users to request their own salary as well as the salary of their direct subordinates.
+
+**example.rego**:
+
+```live:example:module:openable
+package httpapi.authz
+
+# bob is alice's manager, and betty is charlie's.
+subordinates = {"alice": [], "charlie": [], "bob": ["alice"], "betty": ["charlie"]}
+
+default allow = false
+
+# Allow users to get their own salaries.
+allow {
+  some username
+  input.method == "GET"
+  input.path = ["finance", "salary", username]
+  input.user == username
+}
+
+# Allow managers to get their subordinates' salaries.
+allow {
+  some username
+  input.method == "GET"
+  input.path = ["finance", "salary", username]
+  subordinates[input.user][_] == username
+}
+```
+
+Then, create a `bundles` directory into which we'll build a bundle:
+
+```shell
+mkdir bundles
+opa build --output bundles/bundle.tar.gz example.rego
```

Comment:
Any reason not to just write out the bundle to the CWD? I like the simplicity of `opa build example.rego` :D

Reply:
The idea was that `bundles` could then be put in .gitignore .. but thinking about it one might just put bundle.tar.gz there as long as no more bundles are involved. I'll change as I too like that simplicity.

Reply:
Done both here and in contrib.